web-api/src/models.rs

```diff
@@ -219,19 +223,17 @@ impl DocumentProperty {
                         });
                     };
                     trim(string);
-                    if !is_valid_string(string, 2_048) {
+                    let Ok(()) = check_string_property_value(string) else {
```

Comment:
:wrench: a `if is_err()` seems to transport the intention a bit better than `let Ok(())`, with the reused `validate_string()` we can write it as
```suggestion
                if validate_string(string).is_err() {
```